flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/table/OverWindowITCase.scala

```diff
@@ -147,7 +147,7 @@ class OverWindowITCase extends StreamingWithStateTestBase {
   }
 
   @Test
-  def testProcTimeBoundedPartitionedRangeOver(): Unit = {
+  def testProcTimeBoundedPartitionedRowsOver(): Unit = {
```

Comment:
Someone else will have to verify this change, but the other changes look good to me.